src/jiti.ts

```diff
@@ -80,6 +80,12 @@ export default function createJITI (_filename: string = process.cwd(), opts: JIT
     try { return nativeRequire.resolve(id, options) } catch (e) {}
   }
 
+  let tsConfig = { compilerOptions: {} as any }
+  const tsConfigPath = tryResolve('tsconfig')
```

Comment:
Jiti, by design, acts independently of the environment to have predictive results. We cannot depend on tsconfig.json or babelrc